app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  protect_from_forgery with: :exception
```

Comment:
- [ ] You have done an incredible job adding the migration and fixing the errors. Thanks to your changes now it's possible to get to the login and sign-up pages. But the design of your `login` page is not following the wireframe provided. 

### Your design 
![image](https://github.com/jaimevillegas/rails-blog-app/assets/69077061/37daf47d-cf58-4248-a18e-2f47945df512)

### Wireframe 
![blog_login](https://github.com/jaimevillegas/rails-blog-app/assets/69077061/be349d72-4824-4ecd-9cf1-e4375eeb2c94)

- [Optional] I would suggest making your `Login` route your `root` path, this makes it easy for the user to sign in. I had to check out your routes to find the route for the login page.